evalml/model_understanding/graphs.py

```diff
@@ -890,3 +891,59 @@ def graph_prediction_vs_actual_over_time(pipeline, X, y, dates):
                         yaxis={'title': 'Target Values and Predictions'})
 
     return _go.Figure(data=data, layout=layout)
+
+
+def t_sne(X, n_components=2, perplexity=30.0, learning_rate=200.0, metric='euclidean'):
```

Comment:
not sure if necessary, but might be useful to either add that this only supports numeric data, or have an error thrown if a user tries to pass in non-numeric data, just so the error message they get is coming form evalml rather than sklearn.